Dockerfile.base

```diff
@@ -0,0 +1,48 @@
+#logstash-base image, use ci/docker_update_base_image.sh to push updates
+FROM ubuntu:xenial
+
+RUN apt-get update && \
+    apt-get dist-upgrade && \
```

Comment:
@jakelandis don't do `upgrade` in a `Dockerfile`, this just makes things unstable in the long run depending on what happens to the base image. see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get for some details on this

Reply:
good call... i removed the upgrade.